DangProject/Sources/Presentation Layer/Setting/Alarm/ViewModel/AlarmViewModel.swift

```diff
@@ -5,163 +5,211 @@
+        
+        tempAlarmData.append(alarmViewModel)
+        tempAlarmData = tempAlarmData.sorted { $0.time < $1.time }
+        guard let index = self.tempAlarmData.firstIndex(of: alarmViewModel) else { return }
+        addedCellIndex = index
+        resetTotalCellScaleNormal(index: addedCellIndex)
+        alarmDataArrayRelay.accept(tempAlarmData)
+    }
+    
+    func changeIsOnValue(index: Int) {
+        tempAlarmData[index].isOn.toggle()
+        alarmManagerUseCase.changeAlarmNotificationRequest(data: tempAlarmData[index], changedOption: .isOn)
+        alarmDataArrayRelay.accept(tempAlarmData)
+    }
+    
+    func changeUserMessage(index: Int, text: String) {
+        tempAlarmData[index].message = text
+        alarmManagerUseCase.changeAlarmNotificationRequest(data: tempAlarmData[index], changedOption: .message)
+        alarmDataArrayRelay.accept(tempAlarmData)
+    }
+    
+    func changeTime(index: Int, time: Date) {
+        tempAlarmData[index].time = time
+        tempAlarmData[index].timeText = .timeToString(time)
+        tempAlarmData[index].amPm = .timeToAmPm(time)
+        alarmManagerUseCase.changeAlarmNotificationRequest(data: tempAlarmData[index], changedOption: .time)
+        
+        let willChangeAlarmData = tempAlarmData[index]
+        tempAlarmData = tempAlarmData.sorted { $0.time < $1.time }
+        guard let dataIndex = self.tempAlarmData.firstIndex(of: willChangeAlarmData) else { return }
+        changedCellIndex = dataIndex
+        alarmDataArrayRelay.accept(tempAlarmData)
+    }
+    
+    func changeDayOfTheWeek(index: Int, tag: Int) {
+        if tempAlarmData[index].selectedDaysOfWeek.contains(tag) {
+            guard let arrayIndex = tempAlarmData[index].selectedDaysOfWeek.firstIndex(of: tag) else { return }
+            tempAlarmData[index].selectedDaysOfWeek.remove(at: arrayIndex)
         } else {
-            setUpCellStateRelay.accept(.expand(cell))
+            tempAlarmData[index].selectedDaysOfWeek.append(tag)
+            tempAlarmData[index].selectedDaysOfWeek = tempAlarmData[index].selectedDaysOfWeek.sorted()
         }
+        calculateEveryDayAndSelectedDays(index: index)
+        alarmManagerUseCase.changeAlarmNotificationRequest(data: tempAlarmData[index], changedOption: .dayOfWeek)
+        alarmDataArrayRelay.accept(tempAlarmData)
     }
     
-    func branchOutHeightForRow(_ indexPath: IndexPath) -> CGFloat {
-        if cellScaleStateRelay.value == .expand {
-            if selectedIndexRelay.value == indexPath {
-                return UIScreen.main.bounds.maxY/3
-            } else {
-                return UIScreen.main.bounds.maxY/5
-            }
-        } else if cellScaleStateRelay.value == .normal {
-            return UIScreen.main.bounds.maxY/5
+    func changeEveryDay(index: Int) {
+        if tempAlarmData[index].isEveryDay {
+            tempAlarmData[index].scale = .moreExpand
```

Comment:
```suggestion
            tempAlarmData[index].scale = tempAlarmData[index].isEveryDay ? .moreExpand : .expand
```

단순히 할당하는 거 같아서 삼항연산자 제안해봅니다.